toolchest_client/tools/tool.py

```diff
@@ -497,6 +543,12 @@ def run(self):
                 "For support, contact Toolchest with the error log (above) and the following details:\n\n"
                 f"run_id: {pretty_print_run_ids}\n"
             )
+            if self.streaming_enabled:
+                print(
+                    "The error may have occurred during output streaming. "
+                    "Try re-running the same Toolchest command with this argument:\n"
+                    "\tstreaming_enabled=False"
```

Comment:
I think this message degrades the UX because:
- It's the last thing printed, and people read error messages bottom to top. If I saw this, I'd just retry with `streaming_enabled=False`.
- It's distracting. We should only print something in an error message / traceback if it's directly relevant to the error, this would be better in a "troubleshooting" section of the docs.